test/Integration/IntegrationTestBase.cs

```diff
@@ -149,9 +149,14 @@ protected void StartAzurite()
         }
 
         /// <summary>
-        /// This starts the Functions runtime with the specified function.
-        /// Note the functionName is different than the route.
+        /// This starts the Functions runtime with the specified function(s).
         /// </summary>
+        /// <remarks>
+        /// - The functionName is different than its route.<br/>
+        /// - You can start multiple functions by passing in a space-separated list of function names.<br/>
+        /// - The full path to the Functions CLI is required in the ProcessStartInfo because UseShellExecute is set to false.
+        /// We cannot both use shell execute and redirect output at the same time: https://docs.microsoft.com//dotnet/api/system.diagnostics.processstartinfo.redirectstandardoutput#remarks
```

Comment:
I'd just put this and the line above it down in the code - they're just for one part of what this method is doing and isn't going to be something that someone looking at the docs will care about typically.